### PR TITLE
STAC-15459 Fix topic api

### PR DESCRIPTION
**Pull request by julianosk**
### Step 3: Did you add / update tests for your changes in the right area?
- [ ] Unit/Component test
- [ ] Integration test
- [ ] E2E/Molecule test

### Step 4: I'm confident that everything is properly tested:
I got a PO / QA Approval by:
- [ ] Name

### Step 5: Did you add release notes describing the changes you made?
- [ ] Yes

### Step 6: Can we ship this feature to production?
- [ ] Yes, I'm proud of my work. Ship it! :ship:

### CODE REVIEW

File: test/molecule-role/molecule/_shared/install-and-update/install-provisioning-dependencies.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 - name: Install provisioning dependencies (1/2)
   apt:
     name:
-      python3-pip
+      [python3-pip, jq]
```
Comment (TizianoPerrucci) [inline test/molecule-role/molecule/_shared/install-and-update/install-provisioning-dependencies.yml:5]: would we need to still install those 2 deps considering they are already part of the AMI (see receiver.yml) ?

File: test/molecule-role/molecule/_shared/files/docker-compose.base.yml [hunk 2]
```diff
@@ -3,27 +3,34 @@ version: '2.4'
...
       KAFKA_ADVERTISED_HOST_NAME: kafka
       KAFKA_ADVERTISED_PORT: 9092
       KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
       KAFKA_CREATE_TOPICS: "sts_connection_beat_events:1:1,sts_correlate_endpoints:1:1,sts_generic_events:1:1,sts_intake_health:1:1,sts_multi_metrics:1:1,sts_state_events:1:1,sts_topo_agent_integrations:1:1,sts_topology_events:1:1,sts_topo_process_agents:1:1,sts_trace_events:1:1,sts_correlated_connections:1:1"
+      ALLOW_PLAINTEXT_LISTENER: "yes"
+      DOCKER_API_VERSION: 1.22
+      KAFKA_HOME: "/opt/bitnami/kafka"
     volumes:
       - "./verify-or-create-topics.sh:/usr/local/bin/verify-or-create-topics.sh"
+      - /var/run/docker.sock:/var/run/docker.sock
```
Comment (TizianoPerrucci) [inline test/molecule-role/molecule/_shared/files/docker-compose.base.yml:33]: why the kafka container needs a docker socket ?

**Review by TizianoPerrucci — APPROVED** [review]
Good stuff.
2 small questions

File: test/molecule-role/molecule/_shared/install-and-update/install-provisioning-dependencies.yml (hunk 1 above)
Comment (julianosk) [reply test/molecule-role/molecule/_shared/install-and-update/install-provisioning-dependencies.yml:5]: I think this file is used to provision dependencies in other instances (non-receiver), I got an error in one of the builds saying there was no `jq`.

File: test/molecule-role/molecule/_shared/files/docker-compose.base.yml (hunk 2 above)
Comment (julianosk) [reply test/molecule-role/molecule/_shared/files/docker-compose.base.yml:33]: I don't remember where I copied this from, probably it's not needed. I'll remove.

**Pull request merged**